includes/Model/EvaluationResponse/EvaluationResponse.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Arctic\Model\EvaluationResponse;
```

Comment:
I would probably make the namespace `Evaluation`. The idea of the namespace is to group together similar models (e.g., all "evaluation" models within one namespace). The model class can then be something like `Response`. That makes the full class name less redundant.

```
Arctic\Model\Evaluation\Response
```